packages/@o3r-training/showcase-sdk/package.json

```diff
@@ -13,9 +13,6 @@
   "types": "index.d.ts",
   "sideEffects": false,
   "deprecated": "This package is intended for testing purposes only.",
-  "publishConfig": {
```

Comment:
should we add `"private": true` to make sure it's never targeted for publish

Reply:
If it's for that, then yes.